website/docs/docs/dbt-versions/core-versions.md

```diff
@@ -20,7 +35,7 @@ All dbt Core versions released prior to 1.0 and their version-specific documenta
 
 ## EOL version support 
```

Comment:
longer term, i think we should replace this section with a ##backporting policy header and explain our policy there.

Reply:
IMO "backporting" describes an implementation detail (our git branching strategy within the `dbt-core` repo), whereas "end of life" and "ongoing support" are the intended outcome we're looking to achieve & communicate here

There should be a correspondence between "what do we (Core team) choose to backport" and "what types of fixes are included in most recent minor version (active support), versus all pre-EOL versions" — but the target audience for each feels slightly different.